uvm/servlets/admin/config/network/view/DhcpServer.js

```diff
@@ -18,6 +18,23 @@ Ext.define('Ung.config.network.view.DhcpServer', {
             layout: 'border',
             scrollable: true,
             items: [{
+                region: 'north',
+                xtype: 'panel',
+                items: [{
+                    xtype: 'fieldset',
+                    title: 'Global Options'.t(),
+                    // padding: 10,
+                    items:[{
+                        xtype: 'textfield',
+                        fieldLabel: 'Maximum leases'.t(),
+                        labelWidth: 150,
+                        allowBlank: false,
+                        blankText: 'Maxiumum leases must be specified.'.t(),
+                        bind: '{settings.dhcpMaxLeases}',
+                        maskRe: /[0-9]/
```

Comment:
Is the maskRe per character ?  Otherwise should be /[1-9][0-9]*/ ?

Reply:
Yes, it's per character.